apps/sv/frontend/src/__tests__/governance/proposal-details-content.test.tsx

```diff
@@ -96,19 +110,71 @@ const voteResult = {
   ],
 } as ProposalDetailsContentProps;
 
+const queryClient = new QueryClient({
+  defaultOptions: {
+    queries: {
+      refetchInterval: 500,
+      structuralSharing: replaceEqualDeep,
+    },
+  },
+});
+
 const Wrapper: React.FC<{ children: React.ReactNode }> = ({ children }) => {
+  return (
+    <MemoryRouter>
+      <SvConfigProvider>
+        <WrapperProviders children={children} />
+      </SvConfigProvider>
+    </MemoryRouter>
+  );
+};
+
+const WrapperProviders: React.FC<{ children: React.ReactNode }> = ({ children }) => {
+  const config = useSvConfig();
+  const navigate = useNavigate();
+
   return (
     <ThemeProvider theme={theme}>
-      <MemoryRouter>{children}</MemoryRouter>
+      <AuthProvider authConf={config.auth} redirect={(path: string) => navigate(path)}>
```

Comment:
We'll clean this up once the sv ui links are publicly available. For now we create a separate router in the test.